sdk/go/common/apitype/core.go

```diff
@@ -411,6 +413,16 @@ const (
 	VCSRepositoryRootTag StackTagName = "vcs:root"
 )
 
+// ConfigKey is the key for a configuration value. This is just a string, but we use a type alias to provide a richer
+// description of how the string is used in our apitype definitions.
+type ConfigKey = string
+
+const (
+	// PulumiTagsConfigKey sets additional tags for a stack on a deployment. This is additive to any
+	// tags that are already set on the stack.
+	PulumiTagsConfigKey ConfigKey = "pulumi:tags"
```

Comment:
This change added a number of new places where we used this value - so I've consolidated them to one place. The string "pulumi:tags" now appears only here, in the changelog, and in a snapshot test of a project file.